WMIAdventure/frontend/src/js/components/main-menu/atoms/Option/styled-components/OptionHeader.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const OptionHeader = styled.p`

```

Comment:
Może `h1` zamiast `p`?

Reply:
`h1` nie możemy, bo w NavBar'ze jest `h1` jako napis WMI Adventure przy logo

Reply:
To może `h2`? Jakoś tak mi się bardziej wpisuje w semantykę